utils/create-database.js

```diff
@@ -0,0 +1,61 @@
+// utils/create-database.js
```

Comment:
All comments here could be removed, especially the top line with the name of the file! The general purpose of comments is to inform the reader _what_ you are doing rather than _how_ you are doing it, especially in the cases where the code is not able to explain itself succinctly.